austrakka/utils/options.py

```diff
@@ -428,6 +428,18 @@ def opt_hash_check(**attrs: t.Any):
     )
 
 
+def opt_hash_cache(**attrs: t.Any):
+    defaults = {
+        'help': 'Specify whether to use cached hashes after initial check.'
+    }
```

Comment:
Checking that I understand this option also: would a correct more verbose version be
"Specify whether to reuse previous file hashes after initial check. If --no-hash-cache is specified, all existing file hashes will be re-checked for consistency with the server on every sync. If --hash-cache is specified, file hashes will only be checked when they are first downloaded."

Reply:
Yes, but there's more. I'll update the comment to this.

'Specify whether to reuse previous file hashes after '
                'initial check. If --no-hash-cache is specified, all existing '
                'file hashes will be re-checked for consistency with the server '
                'on every sync. If --hash-cache is specified, file hashes will '
                'be checked when they are first downloaded; The successful hashes '
                'are remembered; Future hash checks will compare the server hash'
                'against the locally memorized values. This optimizes the time '
                'it take to hash check large files. When a cache miss is detected '
                'the file is download again and the new hash is memorized locally.'

Reply:
This is the latest (committed) version of the help text.

'Specify whether to reuse previous file hashes after '
                'initial check. If --no-hash-cache is specified, all existing '
                'file hashes will be re-checked for consistency with the server '
                'on every sync. If --hash-cache is specified, file hashes will '
                'be checked when they are first downloaded; the successful hashes '
                'are remembered; future hash checks will compare the server hash '
                'against the locally memorized values. This optimizes the time '
                'it takes to hash check large files on repeat runs. When a cache '
                'miss is detected the file is download again and the new hash is '
                'memorized locally.'